optimum/intel/openvino/modeling_base.py

```diff
@@ -103,6 +104,13 @@ def __init__(
         if enable_compilation:
             self.compile()
 
+    @property
+    def device(self) -> torch.device:
+        """
+        `torch.device`: The device on which the module is (for torch compatibility).
+        """
+        return torch.device(self._device.lower())
```

Comment:
This won't work when device is set to "gpu" / "npu"

Reply:
I guess it makes sense to always return `torch.device("cpu")` then ? since this property is mostly used by `generate` method to check input devices and transformers `pipeline` to put processed inputs on the model's device.